extensions/reactive-datasource/spi/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-reactive-datasource-parent</artifactId>
+        <version>999-SNAPSHOT</version>
```

Comment:
@jmartisk note that we will need to change this one to `3.27.999-SNAPSHOT` when we backport to `3.27`. I will probably remember and worst case CI will catch it!